src/libcpu/src/jit/jit_float.cpp

```diff
@@ -37,14 +37,14 @@ hostHasFMA3()
 }
 
 static void
-truncateToSingleSd(PPCEmuAssembler& a, const PPCEmuAssembler::XmmRegister& reg)
```

Comment:
Commit looks good, but what is the difference between those terms?  In reality, the operations perform the rounding by truncating as far as I could tell (P.S. names were copied from Dolphin's JIT).